problem_builder/templates/html/dashboard.html

```diff
@@ -42,16 +42,26 @@
           {% for mcq in block.mcqs %}
             <tr>
               <th class="desc">{{ mcq.display_name }}</th>
-              <td class="value" {% if mcq.color %}style="border-right-color: {{mcq.color}};"{% endif %}>
-                {% if mcq.value %}{{ mcq.value }}{% endif %}
+              <td class="value"
+                  {% if mcq.color %} style="border-right-color: {{mcq.color}};"{% endif %}
+                  {% if mcq.value and not show_numbers %} aria-label="{{mcq.value}}"{% endif %}
```

Comment:
For accessibility, when "show numbers" is False but there is no value yet, it might be nice to put aria-label="{% trans "No value yet" %}" or something?